src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,95 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip. Requires an OSDisk of >=256GB size and Data disk of >=8TB size"
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:1000gb",
```

Comment:
You can change it to SizeGreaterThan:8000TB if it's an absolute must requirement, so you fail early

Reply:
Updated.